message/src/common/inventory.rs

```diff
@@ -115,10 +115,10 @@ mod tests {
 
 	#[test]
 	fn test_inventory_type_conversion() {
-		assert_eq!(0u32, InventoryType::Error.into());
-		assert_eq!(1u32, InventoryType::MessageTx.into());
-		assert_eq!(2u32, InventoryType::MessageBlock.into());
-		assert_eq!(3u32, InventoryType::MessageFilteredBlock.into());
+		assert_eq!(0u32, u32::from(InventoryType::Error));
```

Comment:
some magic - this change is completely unrelated to all other changes, but I'm getting compilation error:
```
error[E0282]: type annotations needed                                                                                                                                                                          
   --> message/src/common/inventory.rs:118:20                                                                                                                                                                  
    |                                                                                                                                                                                                          
118 |         assert_eq!(0u32, InventoryType::Error.into());                                                                                                                                                   
    |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot infer type for `T` 
```
after this PR (and no compilation errors before)